arch/arm64/boot/dts/qcom/sdm660-xiaomi-lavender-common.dtsi

```diff
@@ -118,6 +141,7 @@
 		reset-gpios = <&tlmm 66 GPIO_ACTIVE_HIGH>;
 		touchscreen-size-x = <1080>;
 		touchscreen-size-y = <2340>;
+		status = "disabled";
 	};
```

Comment:
The whole touchscreen node and it's tlmm-pinctrls (and IRQ 67 pin) should be moved into device-specific DT (`sdm660-xiaomi-lavender-tianma.dts`).

So from tlmm node:
- ts_pins_active
- ts_int_sleep
- ts_rst_sleep